docs/guides/coming-from-git.md

```diff
@@ -0,0 +1,145 @@
+> assumes the reader is at least vaguely familiar with the basic Jujutsu
+> concepts. In addition to this documentation, this
+> [blog post by Stavros](https://www.stavros.io/posts/switch-to-jujutsu-already-a-tutorial/)
+> is also recommended as a resource for familiarizing oneself with the basic
+> concepts.
+
+## Minimal risk
+
+Since Jujutsu is compatible with Git, you can colocate a Jujutsu repository
+and an existing Git repository. This makes adopting Jujutsu very easy as well as
+minimally risky: if something breaks or you want to go back, just delete your
+project's `.jj/` directory.
+
+## jj init for existing Git repositories
+
+Assuming you have [installed](../install-and-setup.md) Jujutsu, you can
+initialize a
+[colocated repository](../git-compatibility/#colocated-jujutsugit-repos)
+in an existing Git repository on your computer like so:
+
+```sh
+cd ~/whatever/project
+jj git init
+```
+
+You can even do this in a Git repository with a dirty working directory.
+
+## Write the code
+
+Write some code or decide it's time for a commit for some of your current
+changes in the working directory.
+
+Write the commit message:
+
+```sh
+jj describe
+```
+
+Awesome! On most `jj` commands, the first thing Jujutsu will do is auto-commit
+all changes (with an empty commit message), so what you just did with
+`jj describe` is to  write this _existing_ commit's message. This commit
+includes all the current changes.
+
+## Splitting instead of committing
+
+You wrote a bunch of code but some of it can't go in that single commit. What
+you do now is:
+
+```sh
+jj split
+```
+
+Now, if you have a terminal without a mouse (or have mouse completely disabled
+in your terminal because it constantly gets in the way, like I do) this next
+screen might confuse you. Interestingly, it was the reason I gave up on jj the
+first time. Here's how this "interactive" screen works:
+
+1. Use the arrows to go up and down the file list
+1. Hit space to select the files that will go to the first commit
+1. Press `c` to confirm your selection
+1. Change the commit message for that first commit, and save
+1. Change the commit message for that second commit, and save
+
+Of course, for that second commit, you might decide to split again later. That's
+part of the game.
+
+## Time to push
+
+Ok, work done, how to push?
+
+We don't push branches in Jujutsu as we do in Git. Instead, we push `bookmarks`.
+Branches are like tree branches: they fork off `main` and grow. Bookmarks are
+like book _marks_: you constantly grow the book and at some points you put
+bookmarks. This means that in Git every new commit is automatically part of a
+growing branch. In Jujutsu, every new commit is part of the whole book and
+nothing else. If you want it to be part of a Git branch, you assign a bookmark
+to it:
+
+```sh
+jj bookmark set fix-login -r @
+# this means I set a new bookmark in the latest/current commit (signified by @)
```

Comment:
nit: In both this example and the next one you use `bookmark set` which has upserting behavior if you mistype the bookmark name. It should be `jj bookmark move <name>` which avoids this failure mode.

Reply:
I had the thought to use `jj bookmark create` for the first one. I'll do that and then use `move` for the second example.